src/en/readallcomicscom/src/eu/kanade/tachiyomi/extension/en/readallcomicscom/ReadAllComics.kt

```diff
@@ -0,0 +1,185 @@
+package eu.kanade.tachiyomi.extension.en.readallcomicscom
+
+import eu.kanade.tachiyomi.network.GET
+import eu.kanade.tachiyomi.network.asObservableSuccess
+import eu.kanade.tachiyomi.network.interceptor.rateLimit
+import eu.kanade.tachiyomi.source.model.FilterList
+import eu.kanade.tachiyomi.source.model.MangasPage
+import eu.kanade.tachiyomi.source.model.Page
+import eu.kanade.tachiyomi.source.model.SChapter
+import eu.kanade.tachiyomi.source.model.SManga
+import eu.kanade.tachiyomi.source.online.ParsedHttpSource
+import eu.kanade.tachiyomi.util.asJsoup
+import okhttp3.HttpUrl.Companion.toHttpUrl
+import okhttp3.HttpUrl.Companion.toHttpUrlOrNull
+import okhttp3.Interceptor
+import okhttp3.Request
+import okhttp3.Response
+import org.jsoup.Jsoup
+import org.jsoup.nodes.Document
+import org.jsoup.nodes.Element
+import org.jsoup.select.Elements
+import rx.Observable
+
+class ReadAllComics : ParsedHttpSource() {
+
+    override val name = "ReadAllComics"
+
+    override val baseUrl = "https://readallcomics.com"
+
+    override val lang = "en"
+
+    override val supportsLatest = false
+
+    private lateinit var searchPageElements: Elements
+
+    override val client = network.cloudflareClient.newBuilder()
+        .addInterceptor(::archivedCategoryInterceptor)
+        .rateLimit(2)
```

Comment:
should be fine to remove this

Reply:
I only asked to remove ratelimit, not everything else

Reply:
Is there any reason to keep that Interceptor around? The extension works fine without it, as best I can tell, and the selector the function uses to extract a url (`.description-archive > p > span > a`) doesn't actually correspond to anything I can find on the source website.

Reply:
sure

iirc it was there since the site used to return categories/entries on search, which just said to open this new category/entry.